<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="5" hostname="Webplat78" failures="1" tests="1" name="Default test" time="42.462" errors="0" timestamp="2024-09-11T19:20:54 IST">
  <testcase classname="testCases.TC001_TransactionHistoryTest" name="TC005_generatedPagesAccordingToPageSize" time="29.658">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	cat dropdo is displayed assertTRUE (aepsTab) failed did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
cat dropdo is displayed assertTRUE (aepsTab) failed did not expect to find [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at testCases.TC001_TransactionHistoryTest.TC005_generatedPagesAccordingToPageSize(TC001_TransactionHistoryTest.java:152)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- TC005_generatedPagesAccordingToPageSize -->
  <testcase classname="testCases.TC001_TransactionHistoryTest" name="TC006_aaa" time="0.0">
    <ignored/>
  </testcase> <!-- TC006_aaa -->
  <testcase classname="testCases.TC001_TransactionHistoryTest" name="TC001_navigateToTxnHistoryPage" time="0.0">
    <ignored/>
  </testcase> <!-- TC001_navigateToTxnHistoryPage -->
  <testcase classname="testCases.TC001_TransactionHistoryTest" name="TC004_telecomTabTableDataFetching" time="0.0">
    <ignored/>
  </testcase> <!-- TC004_telecomTabTableDataFetching -->
  <testcase classname="testCases.TC001_TransactionHistoryTest" name="TC003_datePicker" time="0.0">
    <ignored/>
  </testcase> <!-- TC003_datePicker -->
  <testcase classname="testCases.TC001_TransactionHistoryTest" name="TC002_validatePageDataAccordingToPageSize" time="0.0">
    <ignored/>
  </testcase> <!-- TC002_validatePageDataAccordingToPageSize -->
</testsuite> <!-- Default test -->
